### DISCUSSION TITLE
create-remix@latest on windows produces escape characters in cli

### DISCUSSION THREAD

**Discussion by jonrcrowell** [remix-run/remix#2408] [Q&A] [answered]
I'm trying to create a new app using the terminal on a Windows 10 machine.

```sh
$ npx create-remix@latest
npx: installed 2 in 2.167s
💿 Welcome to Remix! Let's get you set up with a new project.

←[32m?←[39m ←[1mWhere would you like to create your app?←[22m←[0m ←[0m←[2m(./my-remix-app) ←[22m←[60D←[60C
```

I believe these are related to chalk attempting to color-code the text.

Is there a way to get this to work properly on windows?

This happens using powershell, cmd.exe, bash for Windows, in Windows Terminal, and cmder. 

This wasn't happening before stacks were released.

Here is a similar issue reported for the angular cli: <https://github.com/angular/angular-cli/issues/6478>

**Comment by kiliman** [comment, marked as answer]
Interesting. This works fine for me on Windows 11.

PowerShell 7.2.1 and Windows Terminal.

<img width="393" alt="image" src="https://user-images.githubusercontent.com/47168/159287840-b2f814a6-e0b4-41f2-af1a-1a05c447c98d.png">

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/47168/159288198-e01f48ea-12ed-4522-8532-5d51abe09fe3.png">
